src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -196,7 +196,14 @@ private void LookupExtensionMembersInSingleBinder(LookupResult result, TypeSymbo
             {
                 // No need for "diagnose" since we discard the lookup results (and associated diagnostic info)
                 // unless the results are good.
-                LookupMembersInClass(tempResult, extension, name, arity, basesBeingResolved,
+                int remainingArity = arity == 0 ? 0 : arity - extension.Arity;
```

Comment:
>int remainingArity = arity == 0 ? 0 : arity - extension.Arity; [](http://example.com/codeflow?start=16&length=62)

It feels like we might pick up properties when ```remainingArity``` is 0, but  ```arity``` is not 0. I think we should prevent that, i.e. look up should not return properties in a situation like that. Therefore, I think we should adjust behavior of ```WrongArity``` instead.